app/forms/candidate_interface/account_recovery_request_form.rb

```diff
@@ -0,0 +1,49 @@
+module CandidateInterface
+  class AccountRecoveryRequestForm
+    include ActiveModel::Model
+
+    attr_accessor :previous_account_email
+    attr_reader :current_candidate, :previous_candidate
+
+    validates :previous_account_email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+    validate :email_different_from_current_candidate, if: -> { previous_candidate.present? }
+
+    def initialize(current_candidate:, previous_account_email: nil)
+      self.previous_account_email = previous_account_email&.downcase&.strip
+      @current_candidate = current_candidate
+    end
+
+    def self.build_from_candidate(candidate)
+      new(
+        current_candidate: candidate,
+        previous_account_email: candidate.account_recovery_request&.previous_account_email,
+      )
+    end
+
+    def save
+      @previous_candidate = Candidate.find_by(email_address: previous_account_email)
+
+      return false unless valid?
+
+      ActiveRecord::Base.transaction do
+        account_recovery_request = current_candidate.create_account_recovery_request(
+          previous_account_email:,
+          code: AccountRecoveryRequest.generate_code,
+        )
+
+        AccountRecoveryMailer.send_code(
```

Comment:
Only send emails to candidates that exist in our system

Reply:
So here is the thing, I did the account recovery thing and tried using my personal email address. And it sent me a code. But there is no candidate in this application with my personal email address. I will try it again and take a video.

Reply:
Yeah, this just sends emails to any valid email address. I'll keep this conversation open to remind myself to only send emails to candidates that exist in our system